src/components/kompas-survey-popup/kompas-survey-popup.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, h, JSX, Prop } from '@stencil/core'
+import remove from '../../../assets/fontawesome-free-5.15.3-web/svgs/solid/remove.svg'
+
+@Component({
+  tag: 'kompas-survey-popup',
+  styleUrl: '../kompas-survey-popup/kompas-survey-popup.css',
+  shadow: true,
+})
+export class KompasSurveyPopUp {
+  @Prop() isShowPopUp: boolean = true
+  @Prop() token: string = 'token'
+  @Prop() surveyUuid: string = 'surveyUuid'
+
+  private closePopUp = (): void => {
+    const expires = new Date()
+    expires.setTime(expires.getTime() + 1 * 24 * 60 * 60 * 1000)
+    const cookie = 'kompas._isSurveyClosed' + '=' + true + ';expires=' + expires.toUTCString() + 'path=/'
+    document.cookie = cookie
+    this.isShowPopUp = false
+  }
+
+  private goSurvey = (): void => {
+    window.location.href = `https://www.kompas.id/survey?token=${this.token}&id=${this.surveyUuid}&next=${window.location.href}`
```

Comment:
ini url nya udah di proveide dari backend, cukup tambahin params next
```suggestion
    window.location.href = `${this.surveyUrl}&next=${window.location.href}`
```